tools/run_tests/performance/scenario_config.py

```diff
@@ -446,6 +446,20 @@ def scenarios(self):
                 server_threads_per_cq=2,
                 categories=inproc_categories + [SCALABLE])
 
+            yield _ping_pong_scenario(
+                'cpp_protobuf_async_unary_qps_unconstrained_multiple_channels_%s'
```

Comment:
What's the difference between the "regular" cpp_protobuf_async_unary_qps_unconstrained scenario and this one?
It seems that you tweaked the number of channels to 8 (any other important changes?)

Problems:
- While it's definitely a nice find that increasing the number of channels improves the throughput, I don't see much point in adding yet another benchmarking scenario (we do have plenty of them, the number for most of them aren't really being tracked and they are not displayed in the dashboard) - with the current state, this is basically just a way to make the benchmarks take one bit longer.
- why 8 channels and async_server_threads=8? It seems oddly specific. Is this hand tuned for 8core machines? Keep in mind that for C++ we also have 32-core machines and in some sense they are more interesting because they show the scalability bottlenecks.

A few things to investigate:
- It would be good to see if this leads to improvements with 32 core setup as well (which is where C++ is performing worse than Java in our OSS benchmarks and we don't quite know why).
- If setting channels=N in 32-core machines helps, could this performance gain be translated into better throughput with "default" settings but using multiple subchannels internally? (IMHO that should have the same effect as using multiple channels explicitly).
- btw, does C-core automatically use multiple subchannels (=connections) for high throughput scenarios? If so, it might be worth checking if the settings are the same internally and externally. (because we do see quite different numbers in OSS and internally).

CC @vjpai

Reply:
CC @codeblooded

Reply:
During the experiment, I found the issue with the OSS cpp scenario is majorly its configuration. If it works for 8-core, we can perform the same hand tuning for 32-core as well. Cpp folks should have better knowledge to explain the details of its qps worker..

Reply:
CC @paulosjca

Reply:
CC @yashykt

Reply:
> During the experiment, I found the issue with the OSS cpp scenario is majorly its configuration. If it works for 8-core, we can perform the same hand tuning for 32-core as well. Cpp folks should have better knowledge to explain the details of its qps worker..

That's a really good find but since the same scenario config is used for both 8core and 32core workers, this PR as such isn't as useful.
Also, we'd want the default scenario to perform well, not the special one.
The naming "_multiple_channels_" makes no sense because the "default" unconstrained scenario actually uses more channels than this one.

Do you know what is the number of channels that we use for the internal scenario?